src/rules/no-cycle.js

```diff
@@ -26,7 +35,7 @@ module.exports = {
     if (myPath === '<text>') return {} // can't cycle-check a non-file
 
     const options = context.options[0] || {}
-    const maxDepth = options.maxDepth || Infinity
```

Comment:
this means that if you simply don’t set the maxDepth option, it will default to infinity. I don’t think we need an extra schema value to cover this.

Reply:
@ljharb Thank you, yeah I understand the meaning of this conditional assignment.

As you may know, sometimes folks wish to explicitly set options, like here:
https://github.com/airbnb/javascript/blob/7932a52e4b8f5a96089b245e2086769bf30cf0e4/packages/eslint-config-airbnb-base/rules/imports.js#L233-L235